### ISSUE TITLE
Edit EKS guide to specify classic load balancer

### ISSUE THREAD

**Issue by vparekh94** [astronomer/airflow-guides#77]
We've run into issues a few times on enterprise installs on EKS with the ELB that gets provisioned. Until amazon resolves the issue, our guide should be explicit in recommending the classic load balancer instead of the elastic one.

I dont know where in the guide this goes, or if needs any changes in the code outside of the guide, so assigning to @bonominijl

**Comment by schnie** [comment]
We added the NLB annotation back when working with a previous client, getting our first EKS cluster running. After working with AWS support we realized that the NLB is just flat out broken with EKS and that they acknowledge that.

The AWS recommendation was to use the classic ELB, which should be the default when requesting a service of type `LoadBalancer` on EKS.

The relevant and already updated section of the helm configuration is here:
https://github.com/astronomerio/helm.astronomer.io/blob/master/charts/nginx/templates/nginx-service.yaml#L14-L17

This change was made 28 days ago, so anyone who cloned prior to that has the annotation.
https://github.com/astronomerio/helm.astronomer.io/commits/master/charts/nginx/templates/nginx-service.yaml
